### DISCUSSION TITLE
Adding backend to signin page

### DISCUSSION THREAD

**Discussion by Akan45** [c4coderandcreator/Flavour-Fusion#123] [Ideas]
I would like to add backend to signup page so that multiple users can signin and It will be helpful for future login.

Thankyou.

**Comment by c4coderandcreator** [comment]
That's really nice !! you can go for it @Akan45

**Reply by Akan45** [reply]
Thankyou

**Comment by c4coderandcreator** [comment]
can you document the changes with feature update !! It will be helpful in documentation @Akan45  Feel free to discuss your Ideas about this project here or you can drop an email at coderandcreator@gmail.com

**Reply by Akan45** [reply]
yes , of Course, after done with the work I will document the changes.
